LOCALAPP.md

```diff
@@ -159,3 +119,15 @@ This would be required if your app requires a role which none of the test users
 4. Save and close the file
 5. Restart LocalTest
 
+### Tips and tricks
+
+#### Target local webpack-dev-server instead of altinncd.no
+If you're making changes to the [app frontend](https://github.com/Altinn/app-frontend-react/) locally, you're usually
+running the webpack-dev-server on port 8080 (via `yarn start`). There is no need to change the reference back and forth between
+altinncdn.no and localhost in your `App/views/Home/Index.cshtml` for every app you're running - set this
+environment variable before running `docker-compose` and it will instruct the load balancer (nginx) to replace
+references for you on the fly:
+
+```bash
+export SUB_FILTER="sub_filter 'https://altinncdn.no/toolkits/altinn-app-frontend/3/' 'http://localhost:8080/'; sub_filter_once off;"
```

Comment:
This can also be done by uncommenting the relevant line in the `.env` file https://github.com/Altinn/altinn-studio/pull/8086/files#diff-de94efb111b43b11346c372780f8c9dca47c7108ffceb4a7714ed7d367cf3274R20